### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by tranhoaison**
Hi, I fixed some typos in the code.

### CODE REVIEW

File: examples/gno.land/r/demo/banktest/README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ the context of the realm with path "gno.land/r/banktest".
 This means that calls to functions defined within this package
 are encapsulated within this "data realm", where the data is
 mutated based on transactions that can potentially cross many
-realm and non-realm packge boundaries (in the call stack).
+realm and non-realm Package boundaries (in the call stack).
```
Comment (zivkovicmilos) [inline examples/gno.land/r/demo/banktest/README.md:47]: Why capitalize this?

File: gnoland/genesis/genesis_txs.txt [hunk 2]
```diff
@@ -10,7 +10,7 @@
...
...use std.Coins constructors, this isn't generally safe.\n\t\tbanker.SendCoins(pkgaddr, caller, send)\n\t\treturn \"returned!\"\n```\n\nNotice that each realm package has an associated Cosmos address.\n\n\nFinally, the results are rendered via an ABCI query call when you visit [/r/banktest:](/r/banktest:).\n\n```go\nfunc Render(path string) string {\n\t// get realm coins.\n\tbanker := std.GetBanker(std.BankerTypeReadonly)\n\tcoins := banker.GetCoins(std.GetOrigPkgAddr())\n\n\t// render\n\tres := \"\"\n\tres += \"## recent activity\\n\"\n\tres += \"\\n\"\n\tfor _, act := range latest {\n\t\tif act == nil {\n\t\t\tbreak\n\t\t}\n\t\tres += \" * \" + act.String() + \"\\n\"\n\t}\n\tres += \"\\n\"\n\tres += \"## total deposits\\n\"\n\tres += coins.String()\n\treturn res\n}\n```\n\nYou can call this contract yourself, by vistiing [/r/banktest](/r/banktest) and the [quickstart guide](/r/boards:testboard/4).\n"]}],"fee":{"gas_wanted":"2000000","gas_fee":"1000000ugnot"},"signatures":[{"pub_key":{"@type":"/tm.PubKeySecp256k1","value":"A+FhNtsXHjLfSJk1lB8FbiL4mGPjc50Kt81J7EKDnJ2y"},"signature":"iZX/llZlNTdZMLv1goCTgK2bWqzT8enlTq56wMTCpVxJGA0BTvuEM5Nnt9vrnlG6Taqj2GuTrmEnJBkDFTmt9g=="}],"memo":""}
```
Comment (zivkovicmilos) [inline gnoland/genesis/genesis_txs.txt:13]: I would exclude changes to `genesis_txs.txt`, even though they might contain typos in initial transactions

File: gnoland/website/views/package_dir.html [hunk 3]
```diff
@@ -15,7 +15,7 @@
         {{ template "header_buttons" }}
       </div>
 
-      <div id="packge_dir">
+      <div id="Package_dir">
```
Comment (zivkovicmilos) [inline gnoland/website/views/package_dir.html:18]: Why capitalize this?

**Review by zivkovicmilos — APPROVED** [review]
Looks alright 💯 

Thank you for the contribution 🙏 

I've left a few minor comments

File: examples/gno.land/r/demo/banktest/README.md (hunk 1 above)
Comment (tranhoaison) [reply examples/gno.land/r/demo/banktest/README.md:47]: haha, actually that is also my typos. lol

File: gnoland/genesis/genesis_txs.txt (hunk 2 above)
Comment (tranhoaison) [reply gnoland/genesis/genesis_txs.txt:13]: I just realized that I'm using "auto format" on VsCode, which made a lot of difference on the line code.

**Review by moul — CHANGES_REQUESTED** [review]
can you fix the introduced typos pointed-out by @zivkovicmilos?

**Pull request closed without merging**